hydra/hydra.py

```diff
@@ -49,355 +53,475 @@ def __get_api(self, endpoint, parameters=None, headers=None):
+        if isPublic:
+            payload["caseComment"].update({"isPublic": isPublic})
+        if newStatus:
+            payload["additionalData"].update({"newStatus": newStatus})
+        if newInternalStatus:
+            payload["additionalData"].update({"newInternalStatus": newInternalStatus})
+        if nno:
+            payload["additionalData"].update({"needsNewOwner": nno})
         # Both of these fields are for setting the Resolution status and a brief description of how it was resolved. To be used when setting the newStatus and newInternalStatus to "Closed"
-        if newResolution: payload["additionalData"].update({"newResolution": newResolution})
-        if newResolutionDescription: payload["additionalData"].update({"newResolutionDescription": newResolutionDescription})
-        return self.__put_api('cases/v2/comments',
-                payload=payload)
-
+        if newResolution:
+            payload["additionalData"].update({"newResolution": newResolution})
+        if newResolutionDescription:
+            payload["additionalData"].update(
+                {"newResolutionDescription": newResolutionDescription}
+            )
+        return self.__put_api("cases/v2/comments", payload=payload)
 
     def put_tag(self, case_number, tags=[]):
-        return self.__put_api('cases/{}/tags'.format(case_number),
-                payload={"tags":tags})
-
+        return self.__put_api(
+            "cases/{}/tags".format(case_number), payload={"tags": tags}
+        )
 
     def put_owner(self, case_number, user=""):
-        return self.__put_api("cases/{}/owner".format(case_number),
-                payload=user)
-
-
-    def query_cases(self, status=[], fields=[], accounts=[], cases=[],
-           sbrGroups=[], needsNewOwner=None, severity=[], serviceLevel=[],
-           fts=None, ownerSsousername=[], tags=[]):
-       query_params = {}
-
-       if status: query_params.update({'status': ", ".join(status)})
-       if fields: query_params.update({'fields': ", ".join(fields)})
-       if accounts: query_params.update({'accounts': ", ".join(accounts)})
-       if cases: query_params.update({'case_number': ", ".join(cases)})
-       if sbrGroups: query_params.update({'sbrGroups': ", ".join(sbrGroups)})
-       if needsNewOwner: query_params.update({'needsNewOwner': needsNewOwner})
-       if severity: query_params.update({'severity': ", ".join(severity)})
-       if serviceLevel:
-           query_params.update({'serviceLevel': ", ".join(serviceLevel)})
-       if fts: query_params.update({'fts': fts})
-       if ownerSsousername:
-           query_params.update({'ownerSsousername': ", ".join(ownerSsousername)})
-       if tags:
-           query_params.update({'tags': ", ".join(tags)})
+        return self.__put_api("cases/{}/owner".format(case_number), payload=user)
+
+    @deprecation.deprecated(
+        deprecated_in="0.1",
+        removed_in="1.0",
+        details="Use the search_cases function instead",
+    )
```

Comment:
@davemulford you may have to help us learn to start versioning our pushes.

Reply:
That reminds me, I actually meant to update the version in the setup.py to 0.1

Reply:
@sferich888 is there something that you are looking for here before we merge?

Reply:
No, I may just need to be more deliberate about updates.